src/main/java/wooteco/subway/web/member/interceptor/BearerAuthInterceptor.java

```diff
@@ -11,23 +12,25 @@
 
 @Component
 public class BearerAuthInterceptor implements HandlerInterceptor {
+    public static final String BEARER = "bearer";
     private AuthorizationExtractor authExtractor;
     private JwtTokenProvider jwtTokenProvider;
+    private MemberRepository memberRepository;
 
-    public BearerAuthInterceptor(AuthorizationExtractor authExtractor, JwtTokenProvider jwtTokenProvider) {
+    public BearerAuthInterceptor(AuthorizationExtractor authExtractor, JwtTokenProvider jwtTokenProvider, MemberRepository memberRepository) {
         this.authExtractor = authExtractor;
         this.jwtTokenProvider = jwtTokenProvider;
+        this.memberRepository = memberRepository;
     }
 
     @Override
     public boolean preHandle(HttpServletRequest request,
                              HttpServletResponse response, Object handler) {
-        // TODO: Authorization 헤더를 통해 Bearer 값을 추출 (authExtractor.extract() 메서드 활용)
-
-        // TODO: 추출한 토큰값의 유효성 검사 (jwtTokenProvider.validateToken() 메서드 활용)
-
-        // TODO: 추출한 토큰값에서 email 정보 추출 (jwtTokenProvider.getSubject() 메서드 활용)
-        String email = "";
+        String token = authExtractor.extract(request, BEARER);
+        if (!jwtTokenProvider.validateToken(token)) {
```

Comment:
```suggestion
        if (jwtTokenProvider.isInvalidToken(token)) {
```

부정문은 가독성을 해칠 수 있습니다.
Objects.isNull과 Objects.nonNull이 존재하는 이유에 대해 고민해보아요.

Reply:
네 추가적으로 Boolean을 반환하는 메서드명의 기본 규칙이 is로 시작하는 것이라고 공부했습니다. 수정하였습니다.